goalc/compiler/compilation/Asm.cpp

```diff
@@ -311,16 +324,194 @@ Val* Compiler::compile_asm_svf(const goos::Object& form, const goos::Object& res
   return get_none();
 }
 
+// TODO - get rid of duplication
```

Comment:
I'm not too worried about this duplication. It's the price we pay for having that single table of keywords which I think is a net benefit even if you sometimes have stuff like this.